Realm/RLMAsymmetricObject.h

```diff
@@ -72,10 +72,12 @@ NS_ASSUME_NONNULL_BEGIN
 
  @warning This method may only be called during a write transaction.
 
- @param object The asymmetric object to create.
+ @param realm    The Realm to be used to create the asymmetric object..
+ @param value    The value used to populate the object.
 
+ @return  This will return `nil`.
  */
-+ (void)createInRealm:(RLMRealm *)realm withObject:(RLMAsymmetricObject *)object;
++ (instancetype)createInRealm:(RLMRealm *)realm withValue:(id)value __attribute__((warn_unused_result));
```

Comment:
`void` return was correct here.

Reply:
somehow this is causing confusion in the `QueryTests`

```
Aug 15 08:01:32 /Users/realm/workspace/objc_pr/target/osx/xcode_version/13.2.1/Realm/Tests/QueryTests.m:1680:18: error: initializing 'QueryObject *__strong' with an expression of incompatible type 'void'
Aug 15 08:01:32     QueryObject *q1 = [self.queryObjectClass createInRealm:realm withValue:values];
```